common/djangoapps/student/models.py

```diff
@@ -179,6 +179,17 @@ class UserStanding(models.Model):
     standing_last_changed_at = models.DateTimeField(auto_now=True)
 
 
+class UserStandingConfig(ConfigurationModel):
+    disabled = models.TextField(
+        help_text="White-space separated list of usernames of users who should be disabled",
```

Comment:
I'm fine with that. But - perhaps a comment of warning for future civilizations is appropriate? Like:

This was done to fix [a performance problem] and is designed for X users. Likely problems with more users than X:
- Y
- Z